src/pages/CirclesPage/CirclesPage.tsx

```diff
@@ -340,7 +340,15 @@ export const CircleRow = ({ circle, onButtonClick, state }: CircleRowProps) => {
             )}
           </Text>
           {epoch && startDate && endDate && (
-            <Text>{epoch.description ?? `Epoch ${epoch.number}`}</Text>
+            <Text>
+              {epoch.description
```

Comment:
Thanks for the quick fix. I wonder if we could make this into a helper function, or otherwise have this be clearer to read rather than the nested ternarys.... @sshmm

Reply:
done